### PR TITLE
fix: Link to Get Started page in Flutter introduction page

### LINKED ISSUE #172: Broken link in a flutter introduction docs
**Describe the bug**
A link to Get Started Flutter Development page is broken

**To Reproduce**
Go to https://docs.atsign.com/sdk/flutter/introduction/
Try to open Get Started Flutter Development link

**Expected behavior**
I should see the get started page

### PR DESCRIPTION
**Pull request by daniloff200**
closes #172 

**- What I did**
Updated the link

**- How I did it**
By changing path

**- How to verify it**
Just to check the page

**- Description for the changelog**
Fix broken link of Flutter intro page

### CODE REVIEW

**Review by TylerTrott — CHANGES_REQUESTED** [review]
Nice catch on finding the broken link but it looks like your fix does not work. You can test out your code by running 'npm install' and then 'npm run start'. Your redirect currently goes to /sdk/flutter/introduction/docs/start/flutter/

**Comment by daniloff200** [comment]
Hi, @TylerTrott 
Not sure, how it happened. The thing, that you've mentioned was before my fix, and, can be found on https://dev3.atsign.wtf/sdk/flutter/introduction/ 

I've tried my changes locally, and, it works fine.

[Untitled_ Oct 3, 2022 5_34 PM.webm](https://user-images.githubusercontent.com/13692220/193604228-65e79513-8a6c-4618-838d-d3eb7a08487b.webm)

**Comment by TylerTrott** [comment]
> 

Hugo can be a little wonky. I think the easiest fix here is to check what the link address is on the page you want to be on, and simply copy and paste that address into the hyperlink.

File: content/docs/sdk/flutter/introduction.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ Our Dart/Flutter SDK takes full advantage of the atProtocol and implements all t
 
 This is where you can quickly learn and update yourself on various atPlatform concepts.
 
-If you haven't already, we recommend you head to [Get Started Flutter Development](docs/start/flutter/) first.
+If you haven't already, we recommend you head to [Get Started Flutter Development](../../../start/flutter/) first.
```
Comment (xavierchanth) [inline content/docs/sdk/flutter/introduction.md]: Good catch with the URL error, however absolute path is preferred to relative URL. That way if this page moves, the URL will still be valid. `../../../` -> `/docs/`

**Review by xavierchanth — CHANGES_REQUESTED** [review]

File: content/docs/sdk/flutter/introduction.md (hunk 1 above)
Comment (daniloff200) [reply content/docs/sdk/flutter/introduction.md]: @XavierChanth  I've reworked my change, but, with a `/docs` it also does not working

I've made it as `/start/flutter`

**Review by TylerTrott — DISMISSED** [review]
/start/flutter/ works :) Looks good to me

File: content/docs/sdk/flutter/introduction.md (hunk 1 above)
Comment (xavierchanth) [reply content/docs/sdk/flutter/introduction.md]: My mistake, `/start/flutter` is the correct path

**Review by xavierchanth — DISMISSED** [review]

**Comment by daniloff200** [comment]
guys, I have to fix the semantic check, but, dismissed your approvals @TylerTrott  @XavierChanth . Could you do that again, please?

**Review by xavierchanth — APPROVED** [review]

**Comment by JeremyTubongbanua** [comment]
@daniloff200 
Hey! Thank you for your contribution. Your PR was accepted, merged, and it closed an issue. You could be eligible for a prize.

To be eligible, please send an email to [hacktoberfest@atsign.com](mailto:hacktoberfest@atsign.com) with a screenshot of the PR approval email. This is to prove that your email corresponds to the correct GitHub account. Thank you!

When it comes time to distribute the prizes, we will look at all the emails we've received from that email so don't forget to do this!

**Pull request merged**